src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,157 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> tail;
+    private Node<T> head;
+    private int size;
+
+    public MyLinkedList() {
+        head = null;
+        tail = null;
+    }
```

Comment:
these are their default values, so this constructor is not needed, the default one is always generated automatically